### PR TITLE
Smellie dq

### PR DESCRIPTION
**Pull request by Mark--S**
Webpages for SMELLIE Data quality.

### CODE REVIEW

File: bin/mtca_baseline [hunk 1]
```diff
@@ -21,8 +21,6 @@ def PutBaselinesInDatabase(mtc):
 
     baselines = {}
     for name, mtc_name in DBToMTC.iteritems():
-	# mtc server returns baselines in mV so we divide by 1000 to convert
-	# to volts.
```
Comment (tlatorre-uchicago) [inline bin/mtca_baseline]: Why was this deleted? Is this a github bug?

File: minard/views.py [hunk 2]
```diff
@@ -16,6 +16,8 @@
 from math import isnan
 import os
 import sys
+
+
```
Comment (tlatorre-uchicago) [inline minard/views.py]: delete?

File: minard/views.py [hunk 3]
```diff
@@ -772,6 +774,7 @@ def pca_run_detail(run_number):
     
     return render_template('pca_run_detail.html',
                             run_number=run_number)      
+
```
Comment (tlatorre-uchicago) [inline minard/views.py]: delete?

**Comment by tlatorre-uchicago** [comment]
The submenu doesn't seem to work. I would suggest not using a sub drop down menu here, I think the difference between the calibration and physics data quality should be clear by the link names.

File: bin/mtca_baseline (hunk 1 above)
Comment (Mark--S) [inline bin/mtca_baseline]: Yeah I think so.
